codemeta.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@context": "https://w3id.org/codemeta/3.0",
```

Comment:
I think we should also add the version of NESTML here.

Reply:
Cheers, done!